clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -995,6 +1003,98 @@ def CmpOp : CIR_Op<"cmp", [Pure, SameTypeOperands]> {
   let hasVerifier = 0;
 }
 
+//===----------------------------------------------------------------------===//
+// ComplexCreateOp
+//===----------------------------------------------------------------------===//
+
+def ComplexCreateOp : CIR_Op<"complex.create", [Pure, SameTypeOperands]> {
+  let summary = "Create a new complex value from its real and imaginary parts";
+  let description = [{
+    The `cir.complex.create` operation takes two operands of the same type and
```

Comment:
I don't see any use of this operation in the tests, please make sure it shows up.